aiohue/clip.py

```diff
@@ -0,0 +1,32 @@
+class Clip:
+    """Represent Hue clip."""
+
+    def __init__(self, request_2):
+        self._request_2 = request_2
+
+    async def next_events(self):
+        """Note, this method will be pending until next event."""
+        return await self._request_2("get", "eventstream/clip/v2")
+
+    async def resources(self):
+        """Fetch resources from Hue.
+
+        Available types:
+
+        homekit
+        device
+        bridge
+        zigbee_connectivity
+        entertainment
+        light
+        bridge_home
+        grouped_light
+        room
+        scene
+        """
+        return await self._request_2("get", "clip/v2/resource")
+
+    @staticmethod
```

Comment:
Maybe make this a classmethod instead since we return an instance of Clip.

Reply:
```suggestion
    @classmethod
```

Reply:
Will remove it, we don't use it anymore.